datasync/downloader/downloader.go

```diff
@@ -487,14 +496,17 @@ func (d *Downloader) syncWithPeer(p *peerConnection, hash common.Hash, td *big.I
 	}
 
 	fetchers := []func() error{
-		func() error { return d.fetchHeaders(p, origin+1, pivot) }, // Headers are always retrieved
-		func() error { return d.fetchBodies(origin + 1) },          // Bodies are retrieved during normal and fast sync
-		func() error { return d.fetchReceipts(origin + 1) },        // Receipts are retrieved during fast sync
-		func() error { return d.fetchStakingInfos(origin + 1) },    // StakingInfos are retrieved during fast sync
-		func() error { return d.processHeaders(origin+1, pivot, td) },
+		func() error { return d.fetchHeaders(p, origin+1) },     // Headers are always retrieved
```

Comment:
```suggestion
		func() error { return d.fetchHeaders(p, origin + 1) },     // Headers are always retrieved
```

Reply:
This is weird, but this is what `gofumpt` does.

Reply:
got it .yes.